.github/workflows/linux_build.yml

```diff
@@ -0,0 +1,59 @@
+name: Linux Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        swift: ["5.2", "5.1.5"]
```

Comment:
Does "5.2" float?  i.e. - is it 5.2 or does it auto pick up 5.2.1 since they just released that for linux? (and if/when there is a 5.2.2 would it get that)